pkg/registry/input_stream.go

```diff
@@ -147,7 +147,6 @@ func (r *ReplacesInputStream) Next() (*ImageInput, error) {
 
 		// No viable bundle found in the package, can't parse it any further
 		if len(packageErrs) > 0 {
```

Comment:
Do we need this check anymore? Cant we just append the errors regardless?

Reply:
Yup, looks like that would work for the edge cases: https://play.golang.org/p/_SNa9QLWEhQ